### ISSUE TITLE
Why the Dist. to Target grows with the epoch?

### ISSUE THREAD

**Issue by iDTer** [392781/FaceOff#12]
Epoch 28: 
   Loss            = -0.5512599
   Dist. to Image  = 0.9096553
   Dist. to Target = 0.3583954
   Face detection  = True
Epoch 29: 
   Loss            = -0.5638140
   Dist. to Image  = 0.9396414
   Dist. to Target = 0.3758273
   Face detection  = True
Epoch 30: 
   Loss            = -0.5764654
   Dist. to Image  = 0.9016970
   Dist. to Target = 0.3252316
   Face detection  = True

![heath-halle](https://user-images.githubusercontent.com/30014931/187820543-261a77ef-eacb-4896-9da0-72bae1fb8d1c.png)

I feel like the end result is blending the perturbation on the original face image instead of masking it directly. And I found that as the epoch increases, the perturbations appear more and more sparse.

**Comment by 392781** [comment]
Yeah, this is an issue that results from RGB values being mapped between [0,1].  I think it's some issue regarding allowing an alpha to be applied to RGB values or something of that sort. I am not actively developing this so if you pin-point the specific problem, feel free to either comment here or make your own pull-request.

**Comment by 392781** [comment]
I am happy to bounce ideas back and forth with you too!
